src/shared/components/Resources/ResourceItem.tsx

```diff
@@ -19,6 +20,16 @@ const ResourceListItem: React.FunctionComponent<ResourceItemProps> = props => {
   const { resource, index, onClick = () => {} } = props;
   const containerRef = React.createRef<HTMLDivElement>();
 
+  let Preview = null;
+  if (
```

Comment:
should be a separate function hasDisplayableImage() or something similar

Reply:
good idea :)